package.json

```diff
@@ -13,15 +13,16 @@
     "lint": "eslint ./ --ignore-path .gitignore",
     "postinstall": "touch secrets.js",
     "seed": "node script/seed.js",
-    "start": "node server",
+    "start": " webpack -w & nodemon server",
```

Comment:
You can certainly keep this, but our start-dev script is already basically doing these steps... and we'll eventually want to change our start script back to just `node server`, as that's what we'll want Heroku to do to start our dyno (virtual machine that will run our server). I recommend keeping the start script as-is, and getting in the habit of using the `start-dev` script while you're developing your app.